brainiak/utils/fmrisim.py

```diff
@@ -1420,6 +1448,11 @@ def _generate_noise_temporal_drift(trs,
     tr_duration : float
         How long in seconds is each volume acqusition
 
+    basis : str
+        What is the basis function for the drift. Could be made of discrete
+        cosines (number of bases scale with duration) or a sine wave with
```

Comment:
You probably meant `tr_duration` by `duration`? And I did not quite understand `with the temporal order`

Reply:
Rewrote this to clarify. The point is that the drift gets more basis functions (more kinks) when the run is longer

Reply:
Thank you so much for the review again, these were fantastic comments and very insightful